frontend/src/scenes/billing/UnsubscribeSurveyModal.tsx

```diff
@@ -56,12 +58,21 @@ export const UnsubscribeSurveyModal = ({
             }
         >
             <div className="flex flex-col gap-3.5">
-                {unsubscribeError && (
+                {unsubscribeError ? (
                     <LemonBanner type="error">
                         <p>
                             {unsubscribeError.detail} {unsubscribeError.link}
                         </p>
                     </LemonBanner>
+                ) : (
+                    <LemonBanner type="warning">
```

Comment:
I don't think this should be a big warning. Just a standard `info` type will be fine.